### PR TITLE
Unique no 3

### PR DESCRIPTION
**Pull request by ishi0301**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description
Unique_no_3

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#988

### CODE REVIEW

**Comment by ishi0301** [comment]
@HarshCasper @iamrajiv @Abhijit2505 @SKAUL05 Please review my code.

File: C-Plus-Plus/other/Unque_no_3.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//UNIQUE NUMBER III
```
Comment (iamrajiv) [inline C-Plus-Plus/other/Unque_no_3.cpp]: @ishi0301 Update all single-line comments in one multiline comment. Something like this
```
/* 
...
*/
```

File: C-Plus-Plus/other/Unque_no_3.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	for(int i=0;i<64;i++)
+	{
+		cnt[i]%=3;
+		ans+=(cnt[i]*p);
+		p=p<<1;
+	}
+	cout<<ans<<endl;
+	return 0;
+}
+
```
Comment (iamrajiv) [inline C-Plus-Plus/other/Unque_no_3.cpp]: @ishi0301 Add sample input-output

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ishi0301 Update the program name as `Unique_Number_III.cpp`. I think you have used Bitmasking so you can add this concept in code comments.

**Comment by ishi0301** [comment]
Made requested changes . Kindly review. @iamrajiv @SKAUL05 @Abhijit2505 @Kajol-Kumari @HarshCasper @atarax665

**Comment by iamrajiv** [comment]
@ishi0301 Resolve the merge conflicts.

File: C-Plus-Plus/README.md [hunk 3]
```diff
@@ -120,4 +120,4 @@ _add list here_
 - [String Rotation based on Queries](other/string_rotation.cpp)
 - [First Missing Positive Number](other/First_Missing_Positive_Number.cpp)
 - [Remove Outermost Parantheses](other/remove_outermost_parenthesis.cpp)
-
+- [Unique_no_3](other/unque_no_3.cpp)
```
Comment (iamrajiv) [inline C-Plus-Plus/README.md]: @ishi0301 Update this to `[Unique_Number_III ](other/Unique_Number_III.cpp )`

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by ishi0301** [comment]
Resolved conflicts. @iamrajiv @HarshCasper @Abhijit2505 @SKAUL05 @Kajol-Kumari @atarax665 .Please review.

File: C-Plus-Plus/README.md [hunk 4]
```diff
@@ -216,4 +216,8 @@ _add list here_
 - [String Rotation based on Queries](other/string_rotation.cpp)
 - [First Missing Positive Number](other/First_Missing_Positive_Number.cpp)
 - [Remove Outermost Parantheses](other/remove_outermost_parenthesis.cpp)
+<<<<<<< HEAD
```
Comment (iamrajiv) [inline C-Plus-Plus/README.md]: @ishi0301 Remove this.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by ishi0301** [comment]
@iamrajiv removed that line

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ishi0301 Format the program like this
```cpp
/*
Unique_Number_III
Given an array that contains all elements occurring 3 times, but one occurs only once. Find that unique element.
*/

#include <iostream>
using namespace std;

int main() {
	int cnt[64] = {0};
	int n, num;
	cin >> n;
	for (int i = 0; i < n; i++) {
		cin >> num;
		int j = 0;
		while (num > 0) {
			int last_bit = (num & 1);
			cnt[j] += last_bit;
			j++;
			num = num >> 1;
		}
	}
	int p = 1;
	int ans = 0;
	for (int i = 0; i < 64; i++) {
		cnt[i] %= 3;
		ans += (cnt[i] * p);
		p = p << 1;
	}
	cout << ans << endl;
	return 0;
}

/*
Input:
7
6 2 5 2 2 6 6
Output:
5
Time Complexity: O(n)
*/
```

**Comment by ishi0301** [comment]
@iamrajiv formatted the code.

**Review by iamrajiv — APPROVED** [review]
@ishi0301 LGTM!

**Comment by ishi0301** [comment]
@SKAUL05 @Abhijit2505 kindly review.

**Review by SKAUL05 — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @ishi0301 for code

**Pull request merged**